src/version.rs

```diff
@@ -34,10 +34,20 @@ pub struct BuildInfo {
     build_status: String,
     git_tag: String,
     pub istio_version: String,
+    crypto_provider: String,
 }
 
 impl BuildInfo {
     pub fn new() -> Self {
+        #[cfg(feature = "tls-aws-lc")]
+        let crypto_provider = "tls-aws-lc".to_string();
+        #[cfg(feature = "tls-ring")]
```

Comment:
For organization, would it make sense to have the provider be static in tls/lib.rs rather than defined here? Like TLS_VERSIONS

https://github.com/istio/ztunnel/blob/903cf079de4a81925fa1debcb1c48ea20f364169/src/tls/lib.rs#L41